samples/sampler/stories/default/Dropdown.js

```diff
@@ -17,11 +18,15 @@ storiesOf('Agate', module)
 			const items = (new Array(itemCount)).fill().map((i, index) => `Option ${index + 1}`);
 
 			return (
-				<div>
+				<div style={{marginLeft: ri.scaleToRem(10)}}>
```

Comment:
With this, the issue would be reproducible with RTL languages.
Please use `margin-start-end` mixin instead.

Reply:
And the amount 12 would be better.